src/client/amended.rs

```diff
@@ -1,4 +1,5 @@
-use std::mem;
+use alloc::string::ToString;
+use core::mem;
 
 use http::{HeaderMap, HeaderName, HeaderValue, Method, Request, Uri, Version};
```

Comment:
is this using the `http` crate from hyper? would you want to try from this branch? https://github.com/hyperium/http/pull/732 otherwise i'm confused how no_std is pulling in a std crate?

Reply:
This might be where it breaks down. The http crate working in webasm is central to this effort.